packages/examples/packages/invoke-snap/packages/bitcoin-signer/src/index.ts

```diff
@@ -0,0 +1,82 @@
+import { rpcErrors } from '@metamask/rpc-errors';
```

Comment:
Both the Ethereum and Bitcoin example are very similar. I don't think it adds anything to have them both.